problem/models.py

```diff
@@ -105,6 +105,10 @@ class Submission(models.Model):
         (JUDGE_ERROR, 'Judge Error'),
     )
     status = models.CharField(max_length=7, choices=STATUS_CHOICE, default=WAIT)
+    def status_full(self):
```

Comment:
You can try django build-in function 'get_FOO_display'.
https://docs.djangoproject.com/en/1.7/ref/models/instances/#django.db.models.Model.get_FOO_display